doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,227 @@
+
+1. Boot User VM successfully
+#. memory is close the size passed through to User VM
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service VM and User VM can be allocated different IP,
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+VirtIO and bridge/tap is created normally in Service VM
+
+* Test Steps
+
+1. Boot the Device Under Test (DUT), and enter Service VM from the ACRN hypervisor console (serial port).
+
+   .. code-block:: none
+
+      ACRN:\>vm_console
+
+#. Check IP and bridge/tap after log in Service VM.
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot User VM using the ``launch_uos.sh`` and check IP address.
+
+   .. code-block:: none
+   
+      $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+      $ ip a
+
+* Expected Result
+
+IP address should be allocated for both Service VM  and User VM, and ping is ok
+
+
+ST_ACRNIO_SOS_Storage
+*********************
+
+* Description
+
+This test case focus on checking if storage device works correctly in Service VM.
+
+* Prerequisites
+
+storage device is installed
+
+* Test Steps
+
+1. Boot into Service VM
+
+   .. code-block:: none
+  
+   $ fdisk -l 
+
+#. Find storage device and mount it to read/write
+
+   * SATA -> /dev/sda
+   * NVMe -> /dev/nvme0n1
+   * eMMC -> /dev/mmcblk0
+
+* Expected Result
+
+storage device be used normally in Service VM
+
+Graphics Virtualization (GVT)
+*****************************
+
+ST_ACRNGT_SOS_Rendering_Weston_APP
+**********************************
+
+* AcrnGT provide virtual display on Service VM: run weston weston-simple-egl in Service VM,
+
+workload will show normally on Service VM
+
+* Prerequisites
+
+System boot up normally
+
+Install the `IAS Wayland compositor <https://github.com/intel/ias>`_ by installing these bundles
+
+* Test Steps
+
+1. Boot up hypervisor and Service VM
+#. Start weston by command in Service VM
+#. run ./weston-simple-egl in Service VM
+
+   .. code-block:: none
+
+      $ sudo swupd bundle-add software-defined-cockpit x11-server weston-extras
+      $ ps -ef | grep -E 'weston|tty1' | grep -v grep | awk '{print $2}' | xargs kill ; pids=$(lsof |grep tty1 |awk '{print $2}' |uniq -d) ; for pid in $pids; do kill -9 $pid; done;cd /usr/share/xdg/weston/ ; [ -f weston.ini ] && mv weston.ini weston.ini_bak ; cd - &> /dev/null; export XDG_RUNTIME_DIR=/run/user/0;sync;sync;sync;
+      $ weston --tty=1 -i 0 &
+      $ cd /usr/bin
+      $ weston-simple-egl
+
+   .. note::
+
+      do not install bundle desktop, these two may had conflict
+
+* Expected Result
+
+Able to run test apps in Service VM without exception and display normally
+
+TC_ACRNGT_SOS_Rendering_WestonAPP
```

Comment:
Out of curiosity, why does the name of this test start with `TC_` while all others start with `ST_`?

You had an underscore between `weston` and `APP` in the previous test case name, I would recommend using one here for consistency.
```suggestion
TC_ACRNGT_SOS_Rendering_Weston_APP
```